docs/topic-guides/blendable-web-apps/uniform-design-system.md

```diff
@@ -0,0 +1,90 @@
+# Uniform Design System
+
+## Introduction
+
+Uniform design system is a user interface toolkit created by Starcounter. It gives a common look and feel to blendable apps. It is a link between app authors and solution owners.
+
+Unlike many design systems, Uniform is open-ended and themeable. It can be configured, extended or replaced in parts to achieve anything that is possible with HTML, CSS, and JavaScript.
+
+## Mental model of client-side rendering by the Starcounter host
+
+Starcounter host follows a layered design that separates the responsibilities of the app author, the design system and the solution owner. Uniform fits in this design as the middle layer. 
+
+The full design consists of:
+
+1. Native HTML5 and Shadow DOM
+2. Uniform design system
+3. Theme configuration through CSS custom properties
+4. Extend or replace the theme with HTML, CSS, and JavaScript
+
+### Native HTML5 and Shadow DOM
+
+As an app author, you provide the views for your apps with a clear separation of the content and the presentation.
+
+In the content part, you provide the UI components that should be rendered by the Starcounter host as unstyled native HTML5 elements. 
+
+To prevent the Starcounter host from controlling the rendering of a UI component, you can use shadow DOM, preferably in a custom element. The shadow root creates a boundary that blocks the Starcounter host's attempt to apply the look and feel on your components automatically.
```

Comment:
The beginning "To prevent the Starcounter host from controlling" seems disjointed, as you haven't mentioned before, that Starcounter takes over the control of the rendering.

Reply:
Also, this paragraph seems to me like an exceptional case to do advanced things, and it's placed at the beginning before the basics were even briefly described.